lib/qa/authorities/linked_data/config.rb

```diff
@@ -0,0 +1,77 @@
+require 'qa/authorities/linked_data/config/term_config'.freeze
+require 'qa/authorities/linked_data/config/search_config'.freeze
+
+module Qa::Authorities
+  module LinkedData
+    class Config
+      attr_reader :authority_name
+      attr_reader :authority_config
+
+      # Initialize to hold the configuration for the specifed authority.  Configurations are defined in config/authorities/linked_data.  See README for more information.
+      # @param [String] the name of the configuration file for the authority
+      # @return [Qa::Authorities::LinkedData::Config] instance of this class
+      def initialize(auth_name)
+        @authority_name = auth_name
+        auth_config
+      end
+
+      include Qa::Authorities::LinkedData::TermConfig
+      include Qa::Authorities::LinkedData::SearchConfig
+
+      # Return the full configuration for an authority
+      # @return [String] the authority configuration
+      def auth_config
+        @authority_config ||= LINKED_DATA_AUTHORITIES_CONFIG[@authority_name]
+        raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data authority #{@authority_name}" if @authority_config.nil?
+        @authority_config
+      end
+
+      private
+
+        # Removes characters from the query string that are not tolerated by the API
+        #   See oclc sample code at
+        #   http://experimental.worldcat.org/fast/assignfast/js/assignFASTComplete.js
+        def clean_query_string(q)
+          URI.escape(q.gsub(/-|\(|\)|:/, ""))
```

Comment:
I originally copied this from https://github.com/projecthydra-labs/questioning_authority/blob/4cfef296f8ebce5e9c11a350756732c3a8ac3b27/lib/qa/authorities/assign_fast/generic_authority.rb#L52-L57 which was copied from the OCLC js.  I didn't give it much thought.  The exclusions seemed reasonable.  

Now I did a little testing to see what happens.  If I include a `:` or `-` in the query, I get the exact same results.  (In my small test.  It wasn't extensive.)  If I include `(` or `)`, the query fails to process. cURL throws error `-bash: syntax error near unexpected token )`.

Bottom line, I think I agree that this should be removed.  @no-reply, any thoughts?